content/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor.md

```diff
@@ -52,7 +52,7 @@ If the account you want to sponsor does not have a profile on {% data variables.
 
 Before you can sponsor an account, you must have a verified email address. For more information, see "[Verifying your email address](/github/getting-started-with-github/verifying-your-email-address)."
 
-1. On {% data variables.product.product_name %}, navigate to the profile of the user or organization you want to sponsor.
+1. On {% data variables.product.product_name %}, navigate to the profile of the (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile) or (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-organizations-profile) you want to sponsor.
```

Comment:
I don't think this is valid Markdown link syntax, is this what you meant?
```suggestion
1. On {% data variables.product.product_name %}, navigate to the profile of the (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)[user] or (/github/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-organizations-profile)[organization] you want to sponsor.
```

Reply:
Yeah thats what is what i meant. Thanks